sdcm/utils/distro.py

```diff
@@ -95,107 +155,9 @@ def from_os_release(cls, os_release: str):
 
         return distro
 
-    @property
-    def is_unknown(self):
-        return self == self.UNKNOWN
-
-    @property
-    def is_centos7(self):
-        return self == self.CENTOS7
-
-    @property
-    def is_centos8(self):
-        return self == self.CENTOS8
-
-    @property
-    def is_rhel7(self):
-        return self == self.RHEL7
-
-    @property
-    def is_rhel8(self):
-        return self == self.RHEL8
-
-    @property
-    def is_oel7(self):
-        return self == self.OEL7
-
-    @property
-    def is_oel8(self):
-        return self == self.OEL8
-
-    @property
-    def is_amazon2(self):
-        return self == self.AMAZON2
-
-    @property
-    def is_rocky8(self):
-        return self == self.ROCKY8
-
-    @property
-    def is_rhel_like(self):
-        return self.value[0] in ("centos", "rhel", "ol", "amzn", "rocky", )  # pylint: disable=unsubscriptable-object
-
-    @property
-    def is_ubuntu14(self):
-        return self == self.UBUNTU14
-
-    @property
-    def is_ubuntu16(self):
-        return self == self.UBUNTU16
-
-    @property
-    def is_ubuntu18(self):
-        return self == self.UBUNTU18
-
-    @property
-    def is_ubuntu20(self):
-        return self == self.UBUNTU20
-
-    @property
-    def is_ubuntu21(self):
-        return self in (self.UBUNTU21, self.UBUNTU21_10)
-
-    @property
-    def is_ubuntu22(self):
-        return self == self.UBUNTU22
-
-    @property
-    def is_ubuntu(self):
-        return self.value[0] == "ubuntu"  # pylint: disable=unsubscriptable-object
-
-    @property
-    def is_sles(self):
-        return self.value[0] == "sles"  # pylint: disable=unsubscriptable-object
-
-    @property
-    def is_sles15(self):
-        return self == self.SLES15
-
-    @property
-    def is_debian8(self):
-        return self == self.DEBIAN8
-
-    @property
-    def is_debian9(self):
-        return self == self.DEBIAN9
-
-    @property
-    def is_debian10(self):
-        return self == self.DEBIAN10
-
-    @property
-    def is_debian11(self):
-        return self == self.DEBIAN11
-
-    @property
-    def is_debian(self):
-        return self.value[0] == "debian"  # pylint: disable=unsubscriptable-object
-
-    @property
-    def is_debian_like(self):
-        return self.value[0].lower() in ("debian", "ubuntu")  # pylint: disable=unsubscriptable-object
-
     @property
     def uses_systemd(self):
         # Debian uses systemd as a default init system since 8.0, Ubuntu since 15.04, and RedHat-based since 7.0
-        return self not in (self.UNKNOWN, self.UBUNTU14, )
+        return self not in (self.UNKNOWN, self.UBUNTU14_04, )
+
+Distro : enum.Enum = enum.Enum('Distro', enum_data, module= __name__, type = enum_functional_mixin)
```

Comment:
here also is recommended to add a new empty line after this one

Reply:
Done